zap/src/main/java/org/parosproxy/paros/extension/history/LogPanel.java

```diff
@@ -253,6 +259,35 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
         return filterButton;
     }
 
+    private JButton getClearButton() {
+        if (clearButton == null) {
+            clearButton = new JButton();
+            clearButton.setIcon(
+                    new ImageIcon(
+                            LogPanel.class.getResource(
+                                    "/resource/icon/fugue/broom-alert.png")));
```

Comment:
```suggestion
            clearButton.setIcon(DisplayUtils.getScaledIcon(getClass().getResource("/resource/icon/fugue/broom-alert.png")));
```
 and the later `scaleIcon` call is no longer needed.